llvm/test/MC/AsmParser/seh-directive-errors.s

```diff
@@ -33,10 +33,12 @@ f:                                      # @f
 	.seh_stackalloc 32
 	.seh_endprologue
 	nop
+	.seh_startepilogue
```

Comment:
Please add a test somewhere for the ".seh_ directive must appear within an active frame" diagnostic.

Reply:
Added to the `MC/AsmParser/seh-directives-errors.s` test and added checks and tests for ordering between `.seh_endprologue`, `.seh_startepilogue` and `.seh_endepilogue`